src/foam/nanos/cron/ExpandedDayOfWeek.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.ENUM({
+  package: 'foam.nanos.cron',
+  name: 'ExpandedDayOfWeek',
```

Comment:
I would like to use this for Payroll and request
SATURDAY and SUNDAY.   Digital transactions can occur anytime, and also our new etransfers.